src/generate-elements.ts

```diff
@@ -168,12 +168,10 @@ function serializeElementLike(
   const attributes = elementOrMixin.attributes.map(
       (a) => serializeAttribute(elementOrMixin, path, a));
   const properties =
-      elementOrMixin.properties
-          .filter(
-              (p) => !p.private &&
-                  // Blacklist functions until we figure out what to do.
-                  p.type !== 'Function')
+      elementOrMixin.properties.filter((p) => !p.private)
```

Comment:
This is fine for now, since we already filter private members, but I think we actually want to add them to the output and filter in the client.